packages/picasso.js/src/core/chart-components/axis/axis-node-builder.js

```diff
@@ -214,18 +230,30 @@ export default function nodeBuilder(isDiscrete) {
     const layered = state.labels.activeMode === 'layered';
     let majorTickNodes;
 
+    if (settings.isRadial) {
+      buildOpts.startAngle = settings.startAngle;
+      buildOpts.endAngle = settings.endAngle;
+      buildOpts.radius = settings.radius;
+    }
     if (settings.line.show) {
       buildOpts.style = settings.line;
       buildOpts.padding = settings.paddingStart;
-
-      nodes.push(buildLine(buildOpts));
+      if (settings.isRadial) {
+        nodes.push(buildArcLine(buildOpts));
+      } else {
+        nodes.push(buildLine(buildOpts));
+      }
     }
     if (settings.ticks.show) {
       buildOpts.style = settings.ticks;
       buildOpts.tickSize = settings.ticks.tickSize;
       buildOpts.padding = tickSpacing(settings);
-
-      majorTickNodes = tickBuilder(major, buildOpts);
+      buildOpts.tickColor = settings.ticks.tickColor || '#000';
+      if (settings.isRadial) {
+        majorTickNodes = arcTickBuilder(ticks, buildOpts);
```

Comment:
Should we pass in `major` to `arcTickBuilder` as well maybe? The same as we pass in to `tickBuilder`...